search-app/docs/api.md

```diff
@@ -1961,7 +1961,12 @@ Examples of sorting by start_date in descending(Most recent data first) and asce
 
 ### <a name="retrieving-concepts-by-concept-id-and-revision-id"></a> Retrieve concept with a given concept-id or concept-id & revision-id
 
-This allows retrieving the metadata for a single concept. This is only supported for collections and granules. If no format is specified the native format of the metadata will be returned.
+This allows retrieving the metadata for a single concept. This is only supported
```

Comment:
When you generate the docs and look at the HTML do these lines still appear together as a single responsive paragraph or is there a hard break? I thought I saw a problem with artificial line breaks before when I tried to breakup long lines.